<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterWindow</class>
 <widget class="QMainWindow" name="RegisterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Employee Registration</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../icons/favicon.png</normaloff>../icons/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="basicInfo_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>761</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="username_txt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_txt">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>40</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="gmail_txt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>.gmail.com</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="confirm_txt">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Confirm Password</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lname_txt">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>110</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Last Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mname_txt">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>110</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Middle Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="fname_txt">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>First Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="address_txt">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>180</y>
       <width>491</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="birthday_txt">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>230</y>
       <width>131</width>
       <height>33</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Birthday:</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="male_radio">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>230</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Male</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="female_radio">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Female</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="height_txt">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>260</y>
       <width>141</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Height</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="contact_txt">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>260</y>
       <width>191</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Contact Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Username:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>10</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>10</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Confirm Password:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>80</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Middle Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>80</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Gmail:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>230</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>230</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Contact No:</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="generalInfo_frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>521</width>
      <height>281</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="position_combo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Position:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Rate:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="rate_combo">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>151</width>
       <height>33</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Day-Off:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sss_txt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>SSS Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="philhealth_txt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>100</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Philhealth Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pagibig_txt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>PAGIBIG Number</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="tin_txt">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>220</y>
       <width>241</width>
       <height>33</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>TIN Number</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>SSS Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PHILHEALTH Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>PAGIBIG NUMBER:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>200</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TIN:</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_mon">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Monday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_tue">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Tuesday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_wed">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>160</y>
       <width>111</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Wednesday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_thu">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Thursday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_fri">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Friday</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_sat">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>90</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Saturday</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="cancel_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>650</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/close.png</normaloff>../icons/close.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>410</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Proceed to data set capturing</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>430</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>to complete the registration</string>
    </property>
   </widget>
   <widget class="QToolButton" name="train_btn">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>470</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border: None;}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../icons/facerecoglogo.png</normaloff>../icons/facerecoglogo.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Profile Information</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Job Configurations</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>370</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: orangered;</string>
    </property>
    <property name="text">
     <string>Complete the Registration</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>username_txt</tabstop>
  <tabstop>password_txt</tabstop>
  <tabstop>confirm_txt</tabstop>
  <tabstop>fname_txt</tabstop>
  <tabstop>mname_txt</tabstop>
  <tabstop>lname_txt</tabstop>
  <tabstop>gmail_txt</tabstop>
  <tabstop>address_txt</tabstop>
  <tabstop>birthday_txt</tabstop>
  <tabstop>male_radio</tabstop>
  <tabstop>female_radio</tabstop>
  <tabstop>height_txt</tabstop>
  <tabstop>contact_txt</tabstop>
  <tabstop>position_combo</tabstop>
  <tabstop>rate_combo</tabstop>
  <tabstop>checkBox_mon</tabstop>
  <tabstop>checkBox_tue</tabstop>
  <tabstop>checkBox_wed</tabstop>
  <tabstop>checkBox_thu</tabstop>
  <tabstop>checkBox_fri</tabstop>
  <tabstop>checkBox_sat</tabstop>
  <tabstop>sss_txt</tabstop>
  <tabstop>philhealth_txt</tabstop>
  <tabstop>pagibig_txt</tabstop>
  <tabstop>tin_txt</tabstop>
  <tabstop>train_btn</tabstop>
  <tabstop>cancel_btn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
